### ISSUE TITLE
Upgrade progress indikator stuck on "Git Push"

### ISSUE THREAD

**Issue by aguyfromdenmark** [umbraco/Umbraco.Cloud.Issues#646]
### Issue description

When upgrading a project on Umbraco Cloud, the progress bar will be stuck in the "Git Push" state, on the "Verifying homepage HTML" step. 
If you reload the page after having waited on that state for a couple of minutes, the upgrade is already successfully completed. 
I once waited 15 minutes before my patience ran out, and that is when I figured out that this a regular problem. 
I doesn't matter if it's a CMS, Deploy or Forms upgrade. It happens every time, While filing this issue, it still has not progressed from this state. 
![image](https://user-images.githubusercontent.com/6904597/236790943-e6d2f02e-cb6a-4fcf-afa6-e23bb7584d03.png)

Using Firefox 112.0.2.

**Comment by sajumb** [comment]
Hi,
Thank you for your post. The issue that you encounter on this particular project is not a general error but it is likely related to the project, where the internal check does not complete and the page is not updated accordingly. It is most likely a false-positive where the deployment went fine, but the UI fails to update. We will look into the reason for that.

Please open a support ticket and ask support to notify me (Søren, PO at Umbraco Cloud) about the ticket. Or send me the project alias and I will make sure that we look into it. 😃

**Comment by aguyfromdenmark** [comment]
This is a general issue for us, not isolated to this particular project. But I will open a ticket an reference this issue.

**Comment by sajumb** [comment]
We have established that, for individual cloud projects, every now and then the deployment service does not get an expected response within 5 minutes after an upgrade. When this happens, the user, unfortunately, does not receive an update after the "Verifying homepage HTML" message, and the upgrade is not displayed as completed for the user. In the vast majority of cases, the upgrade of the cloud project has been carried out correctly, but the UI experience from the user's point of view is not finished.

We will introduce a message in continuation of the "Verifying homepage HTML" message that the user can simply refresh the page or continue his work if he has waited more than five minutes for an update. Later this year, we will rework the entire UI experience for the longer-term operations such as "Upgrade project", "Add new environment" and "Deployment" so that the user experience will be improved and more informative for the user.

Sorry for the inconvenience. We will close this issue in a few days when the helping text has been released.
